u-service-hello/src/test/java/com/gft/first/microservice_consul_1/MicroserviceConsul1ApplicationTests.java

```diff
@@ -1,17 +1,57 @@
 package com.gft.first.microservice_consul_1;
 
-import org.junit.Ignore;
+import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.mock.web.MockHttpServletResponse;
 import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.context.web.WebAppConfiguration;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
 
 
-@SpringBootTest
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = MicroserviceConsul1Application.class)
+@WebAppConfiguration
 public class MicroserviceConsul1ApplicationTests {
 
+	@Autowired
+	private WebApplicationContext webAppContext;
+
+	private MockMvc mockMvc;
+
+	@Before
+	public void setup() {
+		this.mockMvc = MockMvcBuilders.webAppContextSetup(webAppContext).build();
+	}
+
+	@Test
+	public void testHelloWorldEndpoint() throws Exception {
+		MvcResult result = this.mockMvc.perform(MockMvcRequestBuilders.get("/")).andReturn();
+		MockHttpServletResponse response = result.getResponse();
 
-	public void contextLoads() {
+		assertThat(response.getContentAsString()).contains("Hello");
 	}
 
+	@Test
```

Comment:
If format is optional there should be the test case for also for endpoint without value - please make html default